src/install.sh

```diff
@@ -2,49 +2,49 @@
 
 set -o errexit
 
-KUBECTL=1.18.2
+KUBECTL=$(curl -L -s https://dl.k8s.io/release/stable.txt)
```

Comment:
This change according to https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux

Reply:
This fails in CI with:
```
  /usr/local/bin/kubectl: line 1: syntax error near unexpected token `<'
  /usr/local/bin/kubectl: line 1: `<?xml version='1.0' encoding='UTF-8'?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Details>No such object: kubernetes-release/release/vv1.22.1/bin/linux/amd64/kubectl</Details></Error>'
```